go.mod

```diff
@@ -12,6 +12,8 @@ require (
 	github.com/alecthomas/colour v0.1.0 // indirect
 	github.com/alecthomas/repr v0.0.0-20200325044227-4184120f674c // indirect
 	github.com/alecthomas/units v0.0.0-20210208195552-ff826a37aa15 // indirect
+	github.com/aquasecurity/tracee v0.5.1-0.20210316063448-ec34648b7aff // indirect
+	github.com/aquasecurity/tracee/libbpfgo v0.0.0-20210316063448-ec34648b7aff
```

Comment:
Any reason why you didn't go with https://github.com/cilium/ebpf which appears to be pure Go.

Reply:
I was concerned that this project wouldn't properly/fully reimplement libbpf. There's a lot of low-level fiddling going on there and it seemed more trustworthy to use a library that's maintained by the kernel devs. For example, I need CO-RE but it's unclear how much of it is implemented in there as this issue is still open: https://github.com/cilium/ebpf/issues/114

Reply:
Hi! I've noticed you linked to an issue on our repo. The goal of `cilium/ebpf` is to eventually be at feature parity with libbpf, as libbpf is considered to be the reference implementation for lack of an actual spec. As you've noticed, CO-RE support is not there yet, but there are a few incoming contributions that seek to improve the situation. As to whether we're trustworthy or not, well.. that's entirely up to you. :smile: Note that libbpfgo (the wrapper) is not maintained by kernel devs. If you do happen to give our library a try (outside of CO-RE), please let us know what you find!

Reply:
We're still evaluating which library to use both now and long-term. `cilium/ebpf` would be a good target once eBPF development slows down and things solidify more. `libbpfgo` wrapper is not maintained by the kernel devs but the heavy lifting is done by `libbpf`. We will definitely be keeping an eye on `cilium/ebpf` development.